src/sentry/integrations/slack/utils/users.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any, Mapping, MutableMapping, Sequence
```

Comment:
@mgaeta Ignoring this file for review since I believe it's split from `utils.py`. If there are changes please let me know where.